zkevm/src/prover.rs

```diff
@@ -133,7 +134,7 @@ impl Prover {
     ) {
         let verify_circuit_vk =
             keygen_vk(&self.agg_params, verify_circuit).expect("keygen_vk should not fail");
-
+        println!("come to init_agg_pk_from_verifier_circuit");
```

Comment:
log::info or log::debug 

"enter Prover::init_agg_pk_from_verifier_circuit"

Reply:
update in [c93e757](https://github.com/scroll-tech/common-rs/pull/61/commits/c93e757d6c57a85dac5a45a693651a5f4d11809c) thanks for catching the debug info.